lib/spack/spack/directives.py

```diff
@@ -463,6 +463,8 @@ def _depends_on(pkg, spec, when=None, type=dt.DEFAULT_TYPES, patches=None):
     dep_spec = spack.spec.Spec(spec)
     if not dep_spec.name:
         raise DependencyError("Invalid dependency specification in package '%s':" % pkg.name, spec)
+    if pkg.name == dep_spec.name == "gcc-runtime":
```

Comment:
Dunno how to add a universal dependency *except* for one package other than this hack